zc_install/sql/install/mysql_zencart.sql

```diff
@@ -845,6 +845,29 @@ CREATE TABLE featured (
 
 # --------------------------------------------------------
 
+#
+# Table structure for table 'featured_categories'
+#
+
+DROP TABLE IF EXISTS featured_categories;
+CREATE TABLE featured_categories (
+  featured_categories_id int(11) NOT NULL auto_increment,
+  categories_id int(11) NOT NULL default '0',
```

Comment:
The value's an `int`, its default should also be an int ... i.e. `0` instead of `'0'`.

Reply:
Oops, the `status` field should be similarly changed to `default 1` instead of `default '1'`.